core/new-gui/src/app/workspace/service/undo-redo/undo-redo.service.ts

```diff
@@ -123,4 +130,25 @@ export class UndoRedoService {
   public clearRedoStack(): void {
     this.redoStack = [];
   }
+
+  private sendCommand(update: string): void {
+    if (this.workflowCollabService.getSendData()) {
```

Comment:
this `getSendData` name is confusing. it seems to be a toggle of wether this `sendData` is enabled and returns a boolean right? can we rename it `isSendDataEnabled` (or something else) etc.

Reply:
Changed.